wachter/actions.py

```diff
@@ -8,24 +8,25 @@
 import constants
 import re
 import random
+from typing import *
```

Comment:
Better to import specific names, you will have more control over your code that way (i.e. what if the `typing` module contains `random` as exportable symbol?)